src/views/feed.ecr

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <title><%= @user %> | rethink</title>
```

Comment:
Perhaps changing this in `thoughts.ecr` as well would make sense so it's consistent.